### PR TITLE
feat: MLFlow RDS persistence

### PR DESCRIPTION
**Pull request by LeonLuttenberger**
The MLFlow modules can now use an RDS database for persistence.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: manifests/database-modules.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: mlflow-mysql
+path: git::https://github.com/LeonLuttenberger/idf-modules.git//modules/database/rds?ref=fix/rds-module&depth=1
```
Comment (LeonLuttenberger) [inline manifests/database-modules.yaml:2]: TODO replace this gitpath with the release tag once available.

File: modules/mlflow/mlflow-fargate/app.py [hunk 2]
```diff
@@ -42,6 +42,8 @@ def _param(name: str) -> str:
 artifacts_bucket_name = os.getenv(_param("ARTIFACTS_BUCKET_NAME"))
 lb_access_logs_bucket_name = os.getenv(_param("LB_ACCESS_LOGS_BUCKET_NAME"), DEFAULT_LB_ACCESS_LOGS_BUCKET_NAME)
 lb_access_logs_bucket_prefix = os.getenv(_param("LB_ACCESS_LOGS_BUCKET_PREFIX"), DEFAULT_LB_ACCESS_LOGS_BUCKET_PREFIX)
+rds_hostname = os.getenv(_param("RDS_HOSTNAME"))
+rds_credentials_secret_arn = os.getenv(_param("RDS_CREDENTIALS_SECRET_ARN"))
```
Comment (kukushking) [inline modules/mlflow/mlflow-fargate/app.py:46]: Should we make these optional to allow to switch between RDS/EFS persistence?

Comment (kukushking) [reply modules/mlflow/mlflow-fargate/app.py:46]: + would need to update the image to account for both scenarios.

File: modules/mlflow/mlflow-fargate/deployspec.yaml [hunk 3]
```diff
@@ -1,13 +1,12 @@
+build_type: BUILD_GENERAL1_SMALL
 publishGenericEnvVariables: true
+
 deploy:
   phases:
     install:
       commands:
-      - npm install -g aws-cdk@2.128.0
+      - npm install -g aws-cdk@2.129.0
```
Comment (kukushking) [inline modules/mlflow/mlflow-fargate/deployspec.yaml:8]: Thanks!

File: manifests/database-modules.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+name: mlflow-mysql
```
Comment (kukushking) [inline manifests/database-modules.yaml:1]: Could we use `storage-modules.yaml` for this, please?

**Review by kukushking — APPROVED** [review]
Fantastic, thanks!

**Pull request merged**